infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -7,6 +7,8 @@
 import java.util.List;
 import java.util.Map;
 
+import java.util.UUID;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.UUID'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)